pkg/crc/config/viper_config.go

```diff
@@ -34,6 +38,42 @@ func NewViperStorage(configFile, envPrefix string) (*ViperStorage, error) {
 		return nil, fmt.Errorf("error reading configuration file '%s': %v", configFile, err)
 	}
 	v.WatchConfig()
+	// only watch config file if not running in a terminal/interactively
+	if !terminal.IsTerminal(int(os.Stdout.Fd())) {
+		go func() {
+			var tempConfig = make(map[string]interface{})
+			w, err := fsnotify.NewWatcher()
+			if err != nil {
+				logging.Error(err)
+				return
+			}
+			err = w.Add(configFile)
+			if err != nil {
+				logging.Error(err)
+				return
+			}
+			for {
+				select {
+				case e := <-w.Events:
+					if e.Op == fsnotify.Write {
+						cnf, err := ioutil.ReadFile(configFile)
+						if err != nil {
+							logging.Debug(err)
```

Comment:
Add `continue`. In case of error, cnf might be nil and crash couple of lines after.